backend/src/core/config.ts

```diff
@@ -83,6 +83,39 @@ const config = convict({
       sensitive: true,
     },
   },
+  twilio: {
```

Comment:
Instead of adding new env vars, we can use the demo creds. An example of how to use that [here](https://github.com/opengovsg/postmangovsg/blob/a55db4e1a60ddb5ee76eaf29db83d0440da15802/backend/src/sms/middlewares/sms.middleware.ts#L163-L170)

Reply:
ohh sick, let me do that